wgpu-types/src/lib.rs

```diff
@@ -1656,6 +1674,8 @@ pub struct TextureFormatFeatures {
     pub allowed_usages: TextureUsages,
     /// Additional property flags for the format.
     pub flags: TextureFormatFeatureFlags,
+    /// supported sample count
+    pub sample_count: TextureFormatSampleCountFlags,
```

Comment:
We don't need another bitflags for this - we can use the existing `TextureFormatFeatureFlags::MULTISAMPLE` and split it into 3 flags: `MULTISAMPLE_X2` `MULTISAMPLE_X4` and `MULTISAMPLE_X8` (as x1 multisampling is guarenteed for all formats that you can use at all)

Reply:
That does look cleaner , I'll refactor code to reflect the suggested changes .